### ISSUE TITLE
Click on image inside slide does not trigger click event on iOS

### ISSUE THREAD

**Issue by rico** [gs-shop/vue-slick-carousel#98]
I'm not sure if this is related to https://github.com/kenwheeler/slick/issues/1015.

I have carousel with the following markup for the slides:

```
<div class="image" @click="openGallery">
    <h2>Title</h2>
    <img class="image" src="[url]">
</div>
```
Clicking on the `<h2>` does trigger the click event, but clicking on the `<img>` does not.

Any idea why this happens or what I can do?

**Comment by davidemeroni** [comment]
I have the same issue. Links are working fine, but when they wrap an image they don't work anymore.

**Comment by davidemeroni** [comment]
SOLVED!
`img {
    pointer-events: none;

    @media (hover: hover) {
        pointer-events: auto;
    }
}`
this will prevent any click on the IMG, the on no-touch device you can re-assign the event with a media query. Tested on a few IOS emulator so far.. it work fine.

**Comment by lcabello** [comment]
It works! thanks @davidemeroni !!

**Comment by mattkaye** [comment]
Too bad I didn't see this in time. I ended up bailing on this library just using vue-slick as a nuxt plugin. That fixed the problem for me:

https://github.com/staskjs/vue-slick

Hope this thread is useful for saving others a few hours of research.

**Comment by laksmaria** [comment]
Facing a similar issue here with the `router-link` wrapped with `text` and `image`. However it does work when disable the `touchMove` in the config.

**Comment by laksmaria** [comment]
> SOLVED!
> `img {
> pointer-events: none;
> 
> ```
> @media (hover: hover) {
>     pointer-events: auto;
> }
> ```
> 
> }`
> this will prevent any click on the IMG, then on no-touch device you can re-assign the event with a media query. Tested on a few IOS emulator so far.. it work fine.

Yeah this solution works fine in iOS & Pixel Chrome browser. But for some reason it's not working in Samsung and other android chrome :(
